components/blobserve/leeway.Dockerfile

```diff
@@ -2,9 +2,13 @@
 # Licensed under the GNU Affero General Public License (AGPL).
 # See License-AGPL.txt in the project root for license information.
 
-FROM alpine
-RUN apk add ca-certificates && \
-    adduser -S -D -H -h /app -u 1000 appuser
+FROM alpine:3.13
+
+RUN  apk upgrade \
```

Comment:
nit: maybe a comment that we have to run `apk upgrade` because sources might not be up to date yet would be handy (pretty much what you explained to me in the call :) )